README.md

```diff
@@ -33,6 +33,30 @@ $ docker --help
 
 This should display the list of commands and features.  If it fails with unknown command the installation did not work correctly.
 
+### Several useful facts for working with Docker in Linux
+
+The Docker installation package consists of two independent products which install together: Docker Engine and Docker Desktop.
+  
+It is easy to confuse which system you are currently working with.
+  
+Docker Desktop has a builtin daemon which works in parallel with the `dockerd` daemon.
+  
+Docker Desktop keep it's settings and storage of images and containers in the user's home directory.
```

Comment:
In English, "it's" is a contraction of "it is".  The neuter possessive is "its", no apostrophe.

Also the verb tense here is "keeps", not "keep".

Reply:
Hello @jgiszczak I have fixed these erratas. Take a look here please: https://github.com/AntelopeIO/DUNE/pull/84